### PR TITLE
Uninstall module used on PS 1.6 before using this one

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
### How to test
1. Uninstall both `blockcustomerprivacy` & `ps_dataprivacy` if installed
2. Install [this modified version of `blockcustomerprivacy`](https://github.com/PrestaShop/ps_dataprivacy/files/6266420/blockcustomerprivacy.zip)
 (only the compatibility range has been changed to be able to install it on a 1.7 shop)
3. Install `ps_dataprivacy` with this PR
4. `blockcustomerprivacy` should now be uninstalled and `ps_dataprivacy` should be installed

### CODE REVIEW

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by sarahdib** [comment]
after upgrading from 1.6.1.14 to 1.7.6.5 the module isn't upgrade

File: ps_dataprivacy.php [hunk 1]
```diff
@@ -29,6 +29,11 @@
 
 class Ps_Dataprivacy extends Module
 {
+    /**
+     * @var string Name of the module running on PS 1.6.x. Used for data migration.
+     */
+    const PS_16_EQUIVALENT_MODULE = 'blockcustomerprivacy';
```
Comment (Progi1984) [inline ps_dataprivacy.php:35]:
```suggestion
    private const PS_16_EQUIVALENT_MODULE = 'blockcustomerprivacy';
```

**Review by Progi1984 — APPROVED** [review]

**Comment by Robin-Fischer-PS** [comment]
Hi @atomiix !

Following the steps in the How to Test, I still have both modules installed, blockcustomerprivacy is not unistalled : 

dataprivacy both modules

Thanks :)

**Comment by atomiix** [comment]
Hey @Robin-Fischer-PS, I just tried on my side and it worked 🤔

**Comment by Robin-Fischer-PS** [comment]
Hi @atomiix ,

Thanks for your help. Now it's OK, when I install ps_dataprivacy it uninstall blockcustomerprivacy automatically.

I also did some regression test : 
- Configuration of the module (change message in different language) : :heavy_check_mark: 
- Disable the module : :heavy_check_mark: 
- Enable the module : :heavy_check_mark: 
- Uninstall the module : :heavy_check_mark: 

So it's QA :heavy_check_mark: , thanks !

**Pull request merged**